ts/components/LabelledItem.tsx

```diff
@@ -127,14 +140,16 @@ export const LabelledItem: React.FC<Props> = (props: Props) => {
   const isNotValid = props.isValid === undefined ? false : !props.isValid;
```

Comment:
Wouldn't be more readable expressed as `const isNotValid = !isValid`?

Reply:
@pietro909 I tried but if i pass this, the old logic (not implemented by me) stops working correctly because isNotValid returns true but must return false if props.isValid is undefined